troubleshooting/index.md

```diff
@@ -2,105 +2,78 @@
+      * [Troubleshoot Large Object Map Objects in Ceph Health](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/utility_storage/Troubleshoot_Large_Object_Map_Objects_in_Ceph_Health.md)
+      * [Troubleshoot Failure of RGW Health Check](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/utility_storage/Troubleshoot_RGW_Health_Check_Fail.md)
+ * Troubleshoot Node Management
+      * [Troubleshoot Issues with Redfish Endpoint DiscoveryCheck for Redfish Events from Nodes](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/node_management/Troubleshoot_Issues_with_Redfish_Endpoint_Discovery.md)
+      * [Troubleshoot Interfaces with IP Address Issues](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/node_management/Troubleshoot_Interfaces_with_IP_Address_Issues.md)
+      * [Troubleshoot Loss of Console Connections and Logs on Gigabyte Nodes](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/node_management/Troubleshoot_Loss_of_Console_Connections_and_Logs_on_Gigabyte_Nodes.md)
+ * Troubleshoot Customer Access Network (CAN)
+      * [Troubleshoot CAN Issues](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/network/customer_access_network/Troubleshoot_CAN_Issues.md)
+ * Troubleshoot Dynamic Host Configuration Protocol (DHCP)
+      
+ * Domain Name Service (DNS)
+      * [Troubleshoot Connectivity to Services with External IP addresses](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/network/external_dns/Troubleshoot_Systems_Not_Provisioned_with_External_IP_Addresses.md)
+      * [Troubleshoot DNS Configuration Issues](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/network/external_dns/Troubleshoot_DNS_Configuration_Issues.md)
+ * MetalLB in BGP-Mode
+      * [Troubleshoot Services without an Allocated IP Address](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/network/metallb_bgp/Troubleshoot_Services_without_an_Allocated_IP_Address.md)
+      * [Troubleshoot BGP not Accepting Routes from MetalLB](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/network/metallb_bgp/Troubleshoot_BGP_not_Accepting_Routes_from_MetalLB.md)
+ * Spire
+      * [Restore Spire Postgres without a Backup](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/spire/Restore_Spire_Postgres_without_a_Backup.md)
+      * [Troubleshoot Spire Failing to Start on NCNs](https://github.com/Cray-HPE/docs-csm/blob/release/1.0/operations/spire/Troubleshoot_Spire_Failing_to_Start_on_NCNs.md)
 <a name="known-issues"></a>
 
-## Known Issues
-Listing of known issues and procedures to workaround them in this CSM release.
-
-<a name="known-issues-hardware-discovery"></a>
-
-### Hardware Discovery
-Known issues related to hardware discovery in a system.
- * [HMS Discovery job not creating RedfishEndpoints in Hardware State Manager](known_issues/discovery_job_not_creating_redfish_endpoints.md)
```

Comment:
I don't see this in the list above.  Are you intentionally removing this known issue?

Reply:
This should be there, I added it again